geodetic_utils/src/euler_radians_to_degrees.py

```diff
@@ -1,31 +1,27 @@
 #!/usr/bin/env python
 
-# Simple Python publisher/subscriber node to convert a vector of orientations from degrees to radians
+# Simple Python publisher/subscriber node to convert a vector of orientations
+# from degrees to radians
 
 import rospy
 from geometry_msgs.msg import Vector3
 import numpy
+import sys
 
-def callback_compass(message):
-  degrees = Vector3()
-  degrees.x = numpy.rad2deg(message.x)
-  degrees.y = numpy.rad2deg(message.y)
-  degrees.z = numpy.rad2deg(message.z)
-  degrees_compass_pub.publish(degrees)
-
-def callback_fcu(message):
-  degrees = Vector3()
-  degrees.x = numpy.rad2deg(message.x)
-  degrees.y = numpy.rad2deg(message.y)
-  degrees.z = numpy.rad2deg(message.z)
-  degrees_fcu_pub.publish(degrees)
-
-radians_compass_sub = rospy.Subscriber('/orientation_euler_compass', Vector3, callback_compass)
-radians_fcu_sub = rospy.Subscriber('/orientation_euler_fcu', Vector3, callback_fcu)
-
-degrees_compass_pub = rospy.Publisher('/orientation_euler_compass_degrees', Vector3, queue_size=10)
-degrees_fcu_pub = rospy.Publisher('/orientation_euler_fcu_degrees', Vector3, queue_size=10)
+class eulerDegreestoRadians:
+  
+  def __init__(self):
+    self.radians_sub = rospy.Subscriber(rospy.get_param('~input_topic'), Vector3, self.callback)
+    self.degrees_pub = rospy.Publisher(rospy.get_param('~output_topic'), Vector3, queue_size=10)
 
+  def callback(self, message):
+    degrees = Vector3()
+    degrees.x = numpy.rad2deg(message.x)
```

Comment:
Prefer use standard library functions, like radians() instead of numpy.rad2deg(), when possible. Could also use Python's multiple assignment here (http://stackoverflow.com/questions/5182573/multiple-assignment-semantics).